chef/cookbooks/pacemaker/recipes/default.rb

```diff
@@ -26,18 +26,20 @@
   package pkg
 end
 
-if node[:pacemaker][:setup_hb_gui]
-  node[:pacemaker][:platform][:graphical_packages].each do |pkg|
-    package pkg
-  end
+unless node.platform == 'suse' && node.platform_version.to_f >= 12.0
```

Comment:
This seems a bit convaluted, I am not even sure whether hb_gui is available outside SUSE.